R/dataset.R

```diff
@@ -70,6 +76,19 @@ setMethod("description", "CrunchDataset", function (x) tuple(x)$description)
 setMethod("description<-", "CrunchDataset", setDatasetDescription)
 ##' @rdname describe
 ##' @export
+setMethod("start_date", "CrunchDataset", function (x) tuple(x)$start_date)
```

Comment:
Please run the tests before pushing. Travis found a failure that you easily would have hit because the package doesn't install: https://travis-ci.org/Crunch-io/rcrunch/builds/108167556#L1359